examples/step-37/step-37.cc

```diff
@@ -1158,9 +1158,35 @@ namespace Step37
   // The function that runs the program is very similar to the one in
   // step-16. We do few refinement steps in 3D compared to 2D, but that's
   // it.
+  //
+  // Before we run the program, output some information about the detected
+  // vectorization level. Note that you are likely not getting AVX supported
+  // unless you specify a compiler flag like "-march=native" during
+  // configuration of deal.II.
   template <int dim>
   void LaplaceProblem<dim>::run ()
   {
+    {
+      const unsigned int n_vect_doubles = VectorizedArray<double>::n_array_elements;
+      const unsigned int n_vect_bits = 8*sizeof(double)*n_vect_doubles;
+
+      pcout << "Vectorization over " << n_vect_doubles
+            << " doubles = " << n_vect_bits << " bits (";
+
+      if (n_vect_bits==64)
+        pcout << "disabled";
+      if (n_vect_bits==128)
+        pcout << "SSE2";
+      else if (n_vect_bits==256)
+        pcout << "AVX";
+      else if (n_vect_bits==512)
+        pcout << "AVX512";
+      else
+        AssertThrow(false, ExcNotImplemented());
+
+      pcout << "), VECTORIZATION_LEVEL=" << DEAL_II_COMPILER_VECTORIZATION_LEVEL << std::endl;
+    }
+
```

Comment:
Can you add this also to the output that we put in `results.dox` (section 'program output')?